inference/core/workflows/core_steps/sinks/roboflow/dataset_upload/v1.py

```diff
@@ -139,10 +139,10 @@ class BlockManifest(WorkflowBlockManifest):
         examples=[10, 60],
     )
     max_image_size: Tuple[int, int] = Field(
-        default=(512, 512),
+        default=(1920, 1080),
         description="Maximum size of the image to be registered - bigger images will be "
         "downsized preserving aspect ratio. Format of data: `(width, height)`",
-        examples=[(512, 512), (1920, 1080)],
+        examples=[(1920, 1080), (512, 512)],
     )
     compression_level: int = Field(
         default=75,
```

Comment:
I also ran into this issue, can you also increase the default compression size as part of this PR?

Reply:
Great catch, compression changed to 100.